src/Docfx.App/Helpers/PlaywrightHelper.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices;
+using Docfx.Common;
+using Docfx.Exceptions;
+
+namespace Docfx;
+
+internal static class PlaywrightHelper
+{
+    public static void EnsurePlaywrightNodeJsPath()
+    {
+        // Skip if playwright environment variable exists.
+        if (Environment.GetEnvironmentVariable("PLAYWRIGHT_DRIVER_SEARCH_PATH") != null)
+            return;
+
+        if (Environment.GetEnvironmentVariable("PLAYWRIGHT_NODEJS_PATH") != null)
+            return;
+
+        // Find Node.js executable installation path from PATHs.
+        string exeName = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? "node.exe" : "node";
+
+        var pathEnv = Environment.GetEnvironmentVariable("PATH");
+        if (pathEnv == null)
+            throw new DocfxException("Failed to get `PATH` environment variable.");
+
+        var paths = pathEnv.Split(Path.PathSeparator);
+        foreach (var path in paths)
+        {
+            string fullPath = Path.Combine(path, exeName);
+
+            if (File.Exists(fullPath))
+            {
+                Logger.LogInfo($"Using custom Node.js executable: {fullPath}");
+                Environment.SetEnvironmentVariable("PLAYWRIGHT_NODEJS_PATH", fullPath, EnvironmentVariableTarget.Process);
+                return;
+            }
+        }
+
+        throw new DocfxException("Failed to find Node.js executable from `PATH`.");
```

Comment:
consider add a hint to the error message to install NodeJS or set `PLAYWRIGHT_NODEJS_PATH`

Reply:
I've added error messages in latest commit.
> Node.js executable is not found. Try to install Node.js or set the `PLAYWRIGHT_NODEJS_PATH` environment variable.